python/plugins/processing/algs/gdal/ClipRasterByExtent.py

```diff
@@ -130,13 +130,14 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
         out = self.parameterAsOutputLayer(parameters, self.OUTPUT, context)
         self.setOutputValue(self.OUTPUT, out)
 
-        arguments = [
-            '-projwin',
-            str(bbox.xMinimum()),
-            str(bbox.yMaximum()),
-            str(bbox.xMaximum()),
-            str(bbox.yMinimum()),
-        ]
+        arguments = []
+
+        if not bbox.isNull():
+            arguments.append('-projwin')
+            arguments.append(str(bbox.xMinimum()))
+            arguments.append(str(bbox.yMaximum()))
+            arguments.append(str(bbox.xMaximum()))
+            arguments.append(str(bbox.yMinimum()))
```

Comment:
Even if might not change a lot, use `extend()` instead of multiple `append()` ?
Just a tought